### PR TITLE
update host ports for consistency

### PR DESCRIPTION
**Pull request by erfantkerfan**
ports inside the container are not quoted and ports outside the host were quoted but in some docs, this was not the case,
so it looks like an inconsistency to me and I tried to fix it in this PR.
thanks

### CODE REVIEW

**Review by aroramrinaal — APPROVED** [review]

**Comment by erfantkerfan** [comment]
@aevesdocker @ChrisChinchilla 
this is a simple styling PR, if it does not have any problems please merge it or give me feedback.

File: network/drivers/overlay.md [hunk 1]
```diff
@@ -276,10 +276,10 @@ routing on the individual Docker daemon hosts.
...
 |---------------------------------|-------------------------------------------------------------------------------------------------------------------------------------------------|
-| `-p 8080:80`                    | Map TCP port 80 in the container to port 8080 on the overlay network.                                                                               |
-| `-p 8080:80/udp`                | Map UDP port 80 in the container to port 8080 on the overlay network.                                                                               |
-| `-p 8080:80/sctp`               | Map SCTP port 80 in the container to port 8080 on the overlay network.                                                                              |
-| `-p 8080:80/tcp -p 8080:80/udp` | Map TCP port 80 in the container to TCP port 8080 on the overlay network, and map UDP port 80 in the container to UDP port 8080 on the overlay network. |
+| `-p 8080:80`                    | Map TCP port 80 in the container to port `8080` on the overlay network.                                                                               |
```
Comment (ChrisChinchilla) [inline network/drivers/overlay.md:279]: I am OK with these changes, but for full consistency, should the port 80s here also be in code formatting?

Comment (erfantkerfan) [reply network/drivers/overlay.md:279]: For no apparent reason only ports outside the container were formatted so I only made it uniform without adding extra formatting decisions myself. @ChrisChinchilla

**Comment by ChrisChinchilla** [comment]
Thanks @erfantkerfan !

**Review by ChrisChinchilla — APPROVED** [review]

**Pull request merged**